### ISSUE TITLE
Set a specific time in modifications

### ISSUE THREAD

**Issue by SAOPP** [rianadon/timer-bar-card#21]
It would be great if we could specify a specific amount of time left, for example 10 seconds before the end:

```
                - greater_than_eq: 94
                  active_icon: mdi:timer-10
```
In above example, I find 10 seconds like 94% from my 3 minutes timer :)

**Comment by rianadon** [comment]
Hmm this is a good suggestion.

I predict once I allow for specifying a specific amount of time left, I'll get another issue specifying for a specific amount of time elapsed. I'm going to have to think about how I can support all these cases and not make the configuration schema too crazy.

**Comment by SAOPP** [comment]
Thanks! :)

**Comment by SAOPP** [comment]
Btw, I sat and testing different timers, and for example a timer with five minutes, I couldn't guess the percentage that will be = 10 seconds, 96 is 11 seconds, and 95 is 8 :(

**Comment by rianadon** [comment]
I think fractions should work, if you want to try something like 95.5!

**Comment by SAOPP** [comment]
Oh, thanks homie, I will try this. :+1:

**Comment by SAOPP** [comment]
`- greater_than: 96.5` works for me! :) Timer with 5 minutes - `96.5` it's mean 10 second 'till end. :smiley:

**Comment by SAOPP** [comment]
Btw, just now it dawned on me, I thought that I was picking up icons with percentages equal to 10 seconds, ok, but what if I use different values ​​in automations, and assign a different timer interval depending, for example, on the time of day - then... then, my proposal really has a sense :)

**Comment by rianadon** [comment]
I think I finally have a good syntax for this:

```yaml
modifications:
  - elapsed: 40% # equivalent to the old way of greater_than_eq: 40
    bar_foreground: orange
    active_icon: mdi:fire
    bar_height: 12px
    
  - elapsed: 00:00:50 # at least 50 seconds since the timer started
    bar_foreground: red
    active_icon: mdi:fire-alert

  - remaining: 00:00:10 # when timer has 10 seconds remaining or less
    ...

 - remaining: 1% # when timer has 1% of duration remaining or less
    ...
```

I'm removing `greater_than` but keeping `greater_than_eq` because I don't think the difference is too important stylistically.

**Comment by rianadon** [comment]
Closing now that this is implemented in v1.14! Thank you for the suggestion!

**Comment by tootmoses** [comment]
> I think I finally have a good syntax for this:
>- elapsed: 40% # equivalent to the old way of greater_than_eq: 40
>-bar_foreground: orange
>-active_icon: mdi:fire
>-bar_height: 12px

Hello.
Can I change the color of the icon in these sections?
